### PR TITLE
[5.4] Change how frequencies are tested for schedule

### PR DESCRIPTION
**Pull request by alexbowers**
A change to how schedules are tested for their expression.

I opted to not use mocks, since Cache is not required for the tests to happen.

This, unfortunately, requries PHP 7.0+ to be done in this way, so if you wish to have support for the 5.6+, then this will be required:

```php
$this->event = new Event(
    m::mock('Illuminate\Contracts\Cache\Repository'),
    'php foo'
);
```

If https://github.com/laravel/framework/pull/17084 gets accepted, then this test is required:

```
/**
 * @test
 */
function every_x_minutes()
{
    $this->assertEquals("*/6 * * * * *", $this->event->everyXMinutes(6)->getExpression());
}
```

If https://github.com/laravel/framework/pull/17085 gets accepted, then this test is required:
```
/**
 * @test
 */
function weekends()
{
    $this->assertEquals("* * * * 0,6 *", $this->event->weekends()->getExpression());
}
```

### CODE REVIEW

File: tests/Console/Scheduling/FrequencyTest.php [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            {
+                return $this->expression;
+            }
+        };
+    }
+
+    /**
+     * @test
+     */
+    function every_minute()
```
Comment (GrahamCampbell) [inline tests/Console/Scheduling/FrequencyTest.php]: this does't match the style of our other tests

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Console/Scheduling/FrequencyTest.php (hunk 1 above)
Comment (alexbowers) [inline tests/Console/Scheduling/FrequencyTest.php]: Ah, correct you are. I'll change them to match the format.

Comment (GrahamCampbell) [inline tests/Console/Scheduling/FrequencyTest.php]: Thanks. Not that there's anything wrong with that format, it looks pretty nice, but, just not for laravel/frameworks tests please. :)

Comment (alexbowers) [inline tests/Console/Scheduling/FrequencyTest.php]: Yeah, its my macro for creating tests. I use it for my personal tests, and out of habit used them here. My mistake, should have noticed that :)

**Comment by alexbowers** [comment]
@GrahamCampbell Made that change. Thoughts on https://github.com/laravel/framework/pull/17086/files#diff-ff04bb655c1b51776934f8e56cdeb527R19 ?

**Comment by taylorotwell** [comment]
So these new tests require PHP 7?

**Comment by alexbowers** [comment]
@taylorotwell Not anymore. The original commit did, and I would strongly like to alter how the test works in Laravel 5.5+ when PHP 7.0+ is a requirement to eliminate an unnecessary mock, but for now, it works in PHP 5.6+

**Comment by alexbowers** [comment]
There is a test written for the `weekends` that just got merged in too, I can apply that test tomorrow to this PR.

**Comment by alexbowers** [comment]
@taylorotwell Please can you add in the test in my first message above for `weekends`, or should I submit it as a separate PR?

**Pull request merged**